saltgui/static/stylesheets/dropdown.css

```diff
@@ -17,6 +17,7 @@ header .menu-dropdown-content {
 .run-command-button .menu-dropdown-content {
```

Comment:
I found an even better solution. The main menu (on narrow screens) should always have the size it wants, independent of any other restrictions.
Can you please include the following below this one:
```css
#mainmenumini .run-command-button .menu-dropdown-content {
  max-height: none;
}
```
Note: it should become (almost) the last entry in the file to keep `stylelint` happy.

Reply:
Added. Thx for hint.